src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -106,7 +106,9 @@
 
             // tell our visualforce page we are done loading
             var event = $A.get("e.c:HH_ContainerLoadedEvent");
-            event.fire();
```

Comment:
You have also removed the firing of the event, so there are no "handlers" of the event at all.  I'm okay if you want to keep it, but it seems like a convoluted way to pass an argument to a method to me.